### PR TITLE
Simplify and reorg import docs

### LINKED ISSUE #6772: Import use-cases and behavior documentation

### PR DESCRIPTION
**Pull request by johnnyaug**
Resolves #6772 (part I).

### CODE REVIEW

File: docs/howto/import.md [hunk 1]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+   --from gs://bucket/optional/prefix/ \
+   --to lakefs://my-repo/my-branch/optional/path/
+```
+</div>
+</div>
+
+### Limitations
+
+1. Importing is only possible from the object storage service in which your installation stores its data. For example, if lakeFS is configured to use S3, you cannot import data from Azure.
+2. For security reasons, if you are using lakeFS on top of your local disk, you need to enable the import feature explicitly.
```
Comment (itaiad200) [inline docs/howto/import.md:82]:
```suggestion
2. For security reasons, if you are using lakeFS with the local blockstore, you need to enable the import feature explicitly. 
```

File: docs/howto/import.md [hunk 2]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+</div>
+
+### Limitations
+
+1. Importing is only possible from the object storage service in which your installation stores its data. For example, if lakeFS is configured to use S3, you cannot import data from Azure.
+2. For security reasons, if you are using lakeFS on top of your local disk, you need to enable the import feature explicitly. 
+   To do so, set the `blockstore.local.import_enabled` to `true` and specify the allowed import paths in `blockstore.local.allowed_external_prefixes` (see [configuration reference]({% link reference/configuration.md %})).
+   Since there are some differences between object-stores and file-systems in the way directories/prefixes are treated, local import is allowed only for directories. 
+3. lakeFS cannot manage your metadata if you make changes to data in the original bucket.
+   Making changes to data in the original bucket will not be reflected in lakeFS, and may cause inconsistencies.
```
Comment (itaiad200) [inline docs/howto/import.md:86]:
```suggestion
3. Making changes to data in the original bucket will not be reflected in lakeFS, and may cause inconsistencies. 
```

**Review by itaiad200 — APPROVED** [review]
Much better already!

File: docs/howto/import.md [hunk 3]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
 
-To run import you need the following permissions:
-`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, `fs:ImportFromStorage` and `fs:ImportCancel`. 
-The first 3 permissions are available by default to users in the default Developers group ([RBAC]({% link reference/security/rbac.md %})) or the 
-Writers group ([ACL]({% link reference/security/access-control-lists.md %})). The `fs:Import*` permissions enable the user to import data from any location of the storage 
-provider that lakeFS has access to and cancel the operation if needed. 
-Thus, it's only available to users in group Supers ([ACL]({% link reference/security/access-control-lists.md %})) or SuperUsers([RBAC]({% link reference/security/rbac.md %})).
-RBAC installations can modify policies to add that permission to any group, such as Developers.
+#### Permissions to the storage provider
+The _lakeFS server_ must have permissions to list the objects in the source object store,
```
Comment (arielshaqed) [inline docs/howto/import.md:26]:
```suggestion
The _lakeFS server role_ must have permissions to list the objects in the source object store,
```
?

File: docs/howto/import.md [hunk 4]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+and the source bucket must be in the same region of your destination bucket.
 
+### Using the lakeFS UI
+
+1. In your repository's main page, click the _Import_ button to open the import dialog:
 
-#### lakeFS Permissions
+   ![img.png]({% link assets/img/UI-Import-Dialog.png %})
```
Comment (arielshaqed) [inline docs/howto/import.md:33]:
```suggestion
   ![lakeFS UI import dialog]({% link assets/img/UI-Import-Dialog.png %})
```

File: docs/howto/import.md [hunk 5]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
 
-lakeFS must have permissions to list the objects in the source object store,
-and the source bucket must be in the same region of your destination bucket.  
-In addition, see the following storage provider specific instructions:
+2. Under _Import from_, fill in the location on your object store you would like to import from.
+3. Fill in the import destination in lakeFS 
+4. Add a commit message, and optionally metadata.
+5. Press _Import_
+
+Once the import is complete, the changes are merged into the destination branch.
```
Comment (arielshaqed) [inline docs/howto/import.md:40]: Not sure about the word "merge".

File: docs/howto/import.md (hunk 2 above)
Comment (arielshaqed) [reply docs/howto/import.md:86]: Not scary enough! 👻 
```suggestion
3. Making changes to data in the original bucket will not be reflected in lakeFS, and will cause inconsistencies. 
```

File: docs/howto/import.md [hunk 6]
```diff
@@ -18,22 +18,82 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
...
+3. lakeFS cannot manage your metadata if you make changes to data in the original bucket.
+   Making changes to data in the original bucket will not be reflected in lakeFS, and may cause inconsistencies.
+
+### Permissions required to import data
+
+The lakeFS user running the import command should have the following permissions in lakeFS:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, `fs:ImportFromStorage` and `fs:ImportCancel`.
+The first three permissions above are available by default to the Developers group ([RBAC]({% link reference/security/rbac.md %})) or the
+Writers group ([ACL]({% link reference/security/access-control-lists.md %})).
+The `fs:Import*` permissions are only available to the Supers (ACL) group or the SuperUsers (RBAC) group.
```
Comment (arielshaqed) [inline docs/howto/import.md:94]:
```suggestion
RBAC on lakeFS Cloud and Enterprise allows you fine-grained control of permissions to import.  The lakeFS user running the import command should have the following permissions in lakeFS:
`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, `fs:ImportFromStorage` and `fs:ImportCancel`.
The first three permissions above are available by default to the Developers group ([RBAC]({% link reference/security/rbac.md %})) or the
Writers group ([ACL]({% link reference/security/access-control-lists.md %})).
The `fs:Import*` permissions are only available to the Supers (ACL) group or the SuperUsers (RBAC) group.
```

**Review by arielshaqed — APPROVED** [review]
Very nice!  Shorter and better.

File: docs/howto/import.md (hunk 6 above)
Comment (johnnyaug) [reply docs/howto/import.md:93]: Important note, I'm trying something else

**Pull request merged**